include/nuttx/mutex.h

```diff
@@ -419,7 +419,7 @@ static inline int nxrmutex_trylock(FAR rmutex_t *rmutex)
 
 static inline bool nxrmutex_is_locked(FAR rmutex_t *rmutex)
 {
-  return rmutex->count > 0;
+  return rmutex->holder == gettid();
```

Comment:
This does not implement "is locked" functionality, but rather "is holder".

Reply:
Ok, change to nxrmutex_is_hold